src/main/scala/com/typesafe/sbt/packager/archetypes/systemloader/SystemloaderPlugin.scala

```diff
@@ -158,6 +162,7 @@ object SystemloaderPlugin extends AutoPlugin {
       "stop_runlevels" -> stopRunlevels.getOrElse(""),
       "start_facilities" -> startOn.getOrElse(""),
       "stop_facilities" -> stopOn.getOrElse(""),
+      "file_descriptor_limit" -> fileDescriptorLimit.getOrElse(""),
```

Comment:
If ulimit is called with no arguments it assumes `-f` and will not set limits, instead it will return the value of  `The maximum size of files created by the shell`.

From the man pages: http://linuxcommand.org/man_pages/ulimit1.html
```
 If limit is given, it is the new value of the specified resource
              (the -a option is display only).  If no option is given, then -f
              is  assumed.  Values are in 1024-byte increments, except for -t,
              which is in seconds, -p, which is in units of  512-byte  blocks,
              and  -n and -u, which are unscaled values.  The return status is
              0 unless an invalid option or argument is supplied, or an  error
              occurs while setting a new limit.
```